addOns/pscanrules/src/main/resources/org/zaproxy/zap/extension/pscanrules/resources/Messages.properties

```diff
@@ -16,12 +23,39 @@ pscanrules.charsetmismatch.extrainfo.html.header_metacharset_mismatch=There was
 pscanrules.charsetmismatch.extrainfo.html.metacontenttype_metacharset_mismatch=There was a charset mismatch between the META charset and the META content-type encoding declaration: [{0}] and [{1}] do not match.
 pscanrules.charsetmismatch.extrainfo.xml=There was a charset mismatch between the HTTP Header and the XML encoding declaration: [{0}] and [{1}] do not match.
 
-pscanrules.headerxssprotectionscanner.name = Web Browser XSS Protection Not Enabled
```

Comment:
This change could be reverted.

Reply:
I just noticed the white space change, oops.